moveit_planners/example_planner/CMakeLists.txt

```diff
@@ -0,0 +1,42 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Please remove all references to ``example_planner`` from this PR, and instead we'll add it to moveit_tutorials (as discussed)

Reply:
removed example_planner